js-miniapp-bridge/src/types/custom-permissions.ts

```diff
@@ -2,6 +2,7 @@ export enum CustomPermissionName {
   USER_NAME = 'rakuten.miniapp.user.USER_NAME',
   PROFILE_PHOTO = 'rakuten.miniapp.user.PROFILE_PHOTO',
   CONTACT_LIST = 'rakuten.miniapp.user.CONTACT_LIST',
+  LOCATION = 'rakuten.miniapp.device.LOCATION',
```

Comment:
I think this also needs to be added to the user guide for JS SDK.

Reply:
Yeah, I have updated `js-miniapp-sdk/README.md ` to address this suggestion.